mmdet/datasets/samplers/distributed_sampler.py

```diff
@@ -20,6 +20,8 @@ def __iter__(self):
             indices = torch.arange(len(self.dataset)).tolist()
 
         # add extra samples to make it evenly divisible
+        while len(indices) < self.total_size:
+            indices += indices[:(self.total_size - len(indices))]
```

Comment:
This chunk of code is in conflict (does the same thing) with the one right below. Please leave only one of them. I'd propose to leave the one from upstream.

Reply:
Fixed to the upstream implementation.